itest/rust/src/builtin_tests/script/script_instance_tests.rs

```diff
@@ -222,3 +245,109 @@ impl ScriptInstance for TestScriptInstance {
+    fn remove_named_global_constant(&mut self, _name: StringName) { unreachable!() }
+    fn thread_enter(&mut self) { unreachable!() }
+    fn thread_exit(&mut self) { unreachable!() }
+    fn debug_get_error(&self) -> GString { unreachable!() }
+    fn debug_get_stack_level_count(&self) -> i32 { unreachable!() }
+    fn debug_get_stack_level_line(&self, _level: i32) -> i32 { unreachable!() }
+    fn debug_get_stack_level_function(&self, _level: i32) -> GString { unreachable!() }
+    fn debug_get_stack_level_locals(&mut self, _level: i32, _max_subitems: i32, _max_depth: i32) -> Dictionary { unreachable!() }
+    fn debug_get_stack_level_members(&mut self, _level: i32, _max_subitems: i32, _max_depth: i32) -> Dictionary { unreachable!() }
+    unsafe fn debug_get_stack_level_instance(&mut self, _level: i32) -> *mut c_void { unreachable!() }
+    fn debug_get_globals(&mut self, _max_subitems: i32,_max_depthh: i32) -> Dictionary { unreachable!() }
+    fn debug_parse_stack_level_expression(&mut self, _level: i32, _expression: GString, _max_subitems: i32, _max_depth: i32) -> GString { unreachable!() }
+    fn debug_get_current_stack_info(&mut self) -> Array<Dictionary> { unreachable!() }
+    fn reload_all_scripts(&mut self) { unreachable!() }
+    fn reload_tool_script(&mut self, _script: Option<Gd<Script>>,_soft_reloadd: bool) { unreachable!() }
+    fn get_recognized_extensions(&self) -> godot::prelude::PackedStringArray { unreachable!() }
+    fn get_public_functions(&self) -> Array<Dictionary> { unreachable!() }
+    fn get_public_constants(&self) -> Dictionary { unreachable!() }
+    fn get_public_annotations(&self) -> Array<Dictionary> { unreachable!() }
+    fn profiling_start(&mut self) { unreachable!() }
+    fn profiling_stop(&mut self) { unreachable!() }
+    unsafe fn profiling_get_accumulated_data(&mut self, _info_array: *mut godot::classes::native::ScriptLanguageExtensionProfilingInfo, _info_max: i32) -> i32 { unreachable!() }
+    unsafe fn profiling_get_frame_data(&mut self, _info_array: *mut godot::classes::native::ScriptLanguageExtensionProfilingInfo, _info_max: i32) -> i32 { unreachable!() }
+    fn frame(&mut self) { unreachable!() }
+    fn handles_global_class_type(&self, _type_: GString) -> bool { unreachable!() }
+    fn get_global_class_name(&self, _path: GString) -> Dictionary { unreachable!() }
+    #[cfg(since_api = "4.3")]
+    fn profiling_set_save_native_calls(&mut self, _enable: bool) { unreachable!() }
+    #[cfg(since_api = "4.3")]
+    fn debug_get_stack_level_source(&self, _level: i32) -> GString { unreachable!() }
+    #[cfg(since_api = "4.3")]
+    fn can_make_function(&self) -> bool { unreachable!() }
+    #[cfg(since_api = "4.3")]
+    fn preferred_file_name_casing(&self) -> ScriptNameCasing { unreachable!() }
+    #[cfg(since_api = "4.4")]
+    fn reload_scripts(&mut self, _scripts: Array<Variant>, _soft: bool) { unreachable!() }
+}
+
+// -------- Test Cases --------
```

Comment:
Can you use the usual convention for separators (search for `// ----`), with the "Test cases" on the next line?